src/main/java/core/basesyntax/service/operation/impl/OperationPurchaseHandler.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.operation.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.operation.OperationHandler;
+
+public class OperationPurchaseHandler implements OperationHandler {
+    private static final String SEPARATOR_FOR_LINE_OPERATION_PRODUCTS_QUANTITY = ",";
```

Comment:
Move the constant SEPARATOR_FOR_LINE_OPERATION_PRODUCTS_QUANTITY to a more appropriate location, such as a constants file or the class where it is actually being used for constructing messages.